src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/TrustDetailsValidator.java

```diff
@@ -66,17 +71,69 @@ private boolean validateName(String trustName, Errors errors, String loggingCont
                 && StringValidators.isValidCharacters(trustName, qualifiedFieldName, errors, loggingContext);
     }
 
-    private boolean validateCreationDate(LocalDate dateOfBirth, Errors errors, String loggingContext) {
+    private boolean validateCreationDate(LocalDate creationDate, Errors errors, String loggingContext) {
         String qualifiedFieldName = getQualifiedFieldName(OverseasEntitySubmissionDto.TRUST_DATA, TrustDataDto.CREATION_DATE_FIELD);
 
-        return UtilsValidators.isNotNull(dateOfBirth, qualifiedFieldName, errors, loggingContext)
-                && DateValidators.isDateInPast(dateOfBirth, qualifiedFieldName, errors, loggingContext);
+        return UtilsValidators.isNotNull(creationDate, qualifiedFieldName, errors, loggingContext)
+                && DateValidators.isDateInPast(creationDate, qualifiedFieldName, errors, loggingContext);
+    }
+
+    private void validateCeaseDate(OverseasEntitySubmissionDto overseasEntitySubmissionDto,
+                                   TrustDataDto trustDataDto,
+                                   Errors errors,
+                                   String loggingContext) {
+        final String qualifiedFieldName = getQualifiedFieldName(OverseasEntitySubmissionDto.TRUST_DATA, TrustDataDto.CEASED_DATE_FIELD);
+        final LocalDate trustCeaseDate = trustDataDto.getCeasedDate();
+
+        if (isTrustStillRequired(overseasEntitySubmissionDto, trustDataDto)) {
+            // Cease date of the trust must be 'null' as there are still Individual and/or Corporate BOs associated with this trust
+
+            if (Objects.nonNull(trustCeaseDate)) {
+                final String errorMessage = ValidationMessages.NULL_ERROR_MESSAGE.replace("%s", qualifiedFieldName);
+                setErrorMsgToLocation(errors, qualifiedFieldName, errorMessage);
+                ApiLogger.infoContext(loggingContext, errorMessage);
+            }
+        } else {
+            final LocalDate creationDate = trustDataDto.getCreationDate();
+
+            // Cease date of the trust cannot be 'null' as there are no longer any Individual or Corporate BOs associated with this trust
+
+            if (UtilsValidators.isNotNull(trustCeaseDate, qualifiedFieldName, errors, loggingContext)) {
+                DateValidators.isDateInPast(trustCeaseDate, qualifiedFieldName, errors, loggingContext);
+                DateValidators.isCeasedDateOnOrAfterCreationDate(trustCeaseDate, creationDate, qualifiedFieldName, errors, loggingContext);
+            }
+        }
+    }
+
+    private boolean isTrustStillRequired(OverseasEntitySubmissionDto overseasEntitySubmissionDto, TrustDataDto trustDataDto) {
+        // Use the trust id on TrustDataDto to whizz through all the Individual and Corporate BOs that are associated
+        // with this trust. As soon as a matching BO is found where cease date is null and a trust NOC is set then this
+        // indicates that the trust is still required
+
```

Comment:
you could pull out the trustId null check from both ifs and return early
`if (trustDataDto.getTrustId() == null) {
        return false; // No trust ID, so trust is not required
    }`

Reply:
👍 will implement.

Reply:
Done.